projects/fourier/lib/fourier/services/lint/backbone.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Fourier
+  module Services
+    module Lint
+      class Backbone < Base
```

Comment:
There's a lot of code duplication here. Having all of these nearly-identical classes seems unnecessary. Just call the linter with the appropriate directory from `Commands::Lint`.

Reply:
Part of the duplication has been extracted into `RubocopLinter`. Services were created to have a separation of concerns between commands, which take care of parsing the arguments, and the services, which encapsulate the business logic. If services don't exist we risk coding the logic in the command, which would be the equivalent of placing business logic in views in an MVC pattern.

Reply:
I agree with the general approach. However, in this case there are 3 entities: the Command, the Service, and the Utility. The real business logic is in the Utility, with the Service just being a pass-though with no real logic. So it seems unnecessary to me.

That said, if you really want the Service to remain, I think keeping the Services as `Lint::Swift` and `Lint::Ruby` is a cleaner approach. Or even a single unified Lint service which lints both (which would avoid missing things in the future).